frontends/web/src/locales/en/app.json

```diff
@@ -383,8 +383,8 @@
   "bootloader": {
     "button": "Upgrade firmware now",
     "button_install": "Install firmware now",
-    "progress": "Upgrading: {{progress}}%",
-    "progress_install": "Installing: {{progress}}%",
```

Comment:
this is the reason the lint fails in CI, but keeping for now for testing.

Once approved I'll revert and change in locize. Then we need locize pull